mix.exs

```diff
@@ -34,6 +34,8 @@ defmodule Ecto.Mixfile do
 
   defp deps do
     [{:poolboy, "~> 1.4"},
+     {:sbroker, "~> 0.6", github: "fishcakez/sbroker", branch: "gen"},
```

Comment:
Oops, I meant the Connection module, not sbroker itself. :)

-- 


*José Valimwww.plataformatec.com.br
<http://www.plataformatec.com.br/>Founder and Director of R&D*